apps/settings/js/modules/date_time.js

```diff
@@ -271,7 +265,7 @@ define(function(require) {
     var d = new Date();
     this._time = this._formatTime(d);
 
-    var remainMillisecond = (59 - d.getSeconds()) * 1000;
+    var remainMillisecond = (60 - d.getSeconds()) * 1000;
```

Comment:
I don't know. I wanted to ask @evelynhung. It seems that it's a bug and it caused a worrying behavior - when `_autoUpdateTime` has been fired for the second time (triggered at a random moment), it was fired at, for example, 11:23:59s, and for the whole second, until 11:24:00s it was firing setTimeout as many times as it could - on z3 it was firing it over one thousand times. Then, after this one second it would stop until some other action triggered it again and scheduled it for 59 seconds.

Changing it to 60s fixed that.